src/components/search-lightbox/search-lightbox.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <div>
     <search-input
-      :change="searchTracks"
+      @search-track="searchTracks"
```

Comment:
I think 'search-track' is defining a specific scope. but this is a generic search. so, the I suggest rename it to **change**